.github/workflows/core.yml

```
@@ -697,7 +697,7 @@ jobs:
 
   core_ui_comment:
     name: Post comment with UI diff URLs
-    if: github.event_name == 'pull_request'
+    if: github.event.pull_request.head.repo.full_name == github.repository
```

Not sure if we need to run it on non-PR events:
```suggestion
    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
```